packages/flutter/test/widgets/autocomplete_test.dart

```diff
@@ -883,6 +883,65 @@ void main() {
     expect(find.byKey(optionsKey), findsNothing);
   });
 
+  testWidgets('re-invokes DismissIntent if options not shown', (WidgetTester tester) async {
+    final GlobalKey fieldKey = GlobalKey();
+    final GlobalKey optionsKey = GlobalKey();
+    late FocusNode focusNode;
+    bool wrappingActionInvoked = false;
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: Actions(
+            actions: <Type, Action<Intent>>{
+              DismissIntent: CallbackAction<DismissIntent>(
+                onInvoke: (_) => wrappingActionInvoked = true,
+              ),
+            },
+            child: RawAutocomplete<String>(
+              optionsBuilder: (TextEditingValue textEditingValue) {
+                return kOptions.where((String option) {
+                  return option.contains(textEditingValue.text.toLowerCase());
+                });
+              },
+              fieldViewBuilder: (BuildContext context, TextEditingController fieldTextEditingController, FocusNode fieldFocusNode, VoidCallback onFieldSubmitted) {
+                focusNode = fieldFocusNode;
+                return TextFormField(
+                  key: fieldKey,
+                  focusNode: focusNode,
+                  controller: fieldTextEditingController,
+                  onFieldSubmitted: (String value) {
+                    onFieldSubmitted();
+                  },
+                );
+              },
+              optionsViewBuilder: (BuildContext context, AutocompleteOnSelected<String> onSelected, Iterable<String> options) {
+                return Container(key: optionsKey);
+              },
+            ),
+          ),
+        ),
+      ),
+    );
+
+    // Enter text to show options.
+    focusNode.requestFocus();
+    await tester.enterText(find.byKey(fieldKey), 'ele');
+    await tester.pumpAndSettle();
+    expect(find.byKey(fieldKey), findsOneWidget);
+    expect(find.byKey(optionsKey), findsOneWidget);
+
```

Comment:
Nit: What if you added one more step here for completeness:  Long press the text and assert that the text selection toolbar shows up.  Press escape and assert that it goes away, the options are still visible, and wrappingActionInvoked is false.  Then proceed with the rest of the test.

That should be how it works, right?

Reply:
Doesn't seem within the scope of this test for me. This fix doesn't really affect the text selection toolbar dismiss behavior in an `Autocomplete` so the added asserts wouldn't fail either way.